enrollment-server-onboarding/src/main/resources/application.properties

```diff
@@ -29,6 +29,7 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth
 spring.datasource.username=powerauth
 spring.datasource.password=
 spring.datasource.hikari.auto-commit=false
+spring.datasource.hikari.pool-name=HikariPool-Onboarding-Server
```

Comment:
I suggest we keep the Hikari pool name same as the component name, e.g. `HikariPool-Enrollment-Server-Onboarding`.